lib/roo_on_rails/default.env

```diff
@@ -9,3 +9,8 @@ NEW_RELIC_DEVELOPER_MODE=false
 NEW_RELIC_LICENSE_KEY=override-me
 RACK_SERVICE_TIMEOUT=10
 RACK_WAIT_TIMEOUT=30
+SIDEKIQ_ENABLED=true
+SIDEKIQ_THREADS=5
+SIDEKIQ_DB_POOL_SIZE=6
+SIDEKIQ_DB_REAPING_FREQUENCY=10
```

Comment:
I've juyst taken these directly from LD as a starting point.

Looking into it, `SIDEKIQ_DB_REAPING_FREQUENCY` indirectly sets `DB_REAPING_FREQUENCY` which  in tern sets ActiveRecords reaping_frequency. With large amounts of sidekiq concurrency, as I understand it, you'd want a different value for workers vs the web server.

I'm going to park this ticket until https://github.com/deliveroo/roo_on_rails/pull/10 lands. While we probably don't want to change the default settings now it's important they're configurable and that ticket provides the pluming for setting it up